packages/migrate/src/transforms/2790-fix-transform.ts

```diff
@@ -0,0 +1,206 @@
+import ts from 'typescript';
+
+import path from 'path';
+
+import FixTransform, { FixedFile } from '../interfaces/fix-transform';
+import { findNodeAtPosition, insertStringIntoSourceFile } from '../helpers/typescript-ast';
+import RehearsalService from 'src/rehearsal-service';
+
+const QUESTION_MARK = '?';
+
+function findLocalInterface(
```

Comment:
I think it could make sense to pull out into a separate package. I would love to use these utils for linters and codemods. I do have a hard time believing these utils don't already exist somewhere. Like VScode essentially does this same process for "Go to Definition" and I'm sure linters exist that do the same thing.

Reply:
@brendaHuang perhaps look for a linter or codemod that already exists that will go and find the source of a type in another file?

Reply:
@asakusuma Good point.
@brendaHuang Let's keep helpers in migrate for now, will move the out in separate PR (PPTF-4349).

Reply:
sounds good